### PR TITLE
smart squotes and apostrophe

### PR DESCRIPTION
**Pull request by rwxrob**

### CODE REVIEW

**Comment by rwxrob** [comment]
Hey there, would you mind giving me an update as to the possibility of this being accepted? If it is not something you are interested in I understand. I just want to know if I should reconfigure my lab systems to use my fork or remain with your main one. Thank you.

**Comment by alerque** [comment]
Sorry this is just a big change to review and I haven't been on the right systems to do it. Also next time please make changes in the original branch, opening new PR's makes it hard to follow the history and see what changed between reviews.

That being said I think this looks good overall. I'm sure we'll be able to include it here so no need to fork. There might be a few things to tweak first. Let me try to get it running locally.

File: syntax/pandoc.vim [hunk 1]
```diff
@@ -223,6 +224,14 @@ endif
 
 " Syntax Rules: {{{1
 
+" corrects distracting conceal (ligature) background colors, blue makes 
+" it obvious that the character is a ligature instead of an actual Unicode
+" character since both can exist in the same file
+hi Conceal ctermbg=none ctermfg=Blue
```
Comment (alerque) [inline syntax/pandoc.vim]: Could this be linked to an existing highlight group of some sort rather than hard coding a color? That way it will play nice with themes. `Special`, `Macro`, or `Character` might work.

Comment (alerque) [reply syntax/pandoc.vim]: Actually I think this might be better left to color schemes entirely. I get that the vim defaults are pretty bad, but for any color scheme that _does_ try to set the colors for `Conceal` having a plugin hard code an override for that is kind of rough luck. I think I'd rather see this bit contributed to whatever colorscheme you use and/or added to users' vim-rc file.

**Comment by rwxrob** [comment]
This is not an enhancement. As you will find many of my changes correct omissions in ranges that are expected.

I am closing this ticket and will be maintaining my own fork from now on and  adding several other critical formatting and color fixes to those you have already rejected, as well as several other things that have bothered me for some time:

* Removing distracting, hard to read, often unsupported ligatures
* Fixing the broken table syntax highlighting.
* Adding a theme system for those who need to tweek their colors.

These changes and others constitute a significant diversion from the existing project so I feel forking is best. In fact, because I will be removing a lot of functionality and catering to my simplified (Ezmark) Pandoc variation I'll change the name to `vim-pandoc-syntax-ez` instead. It is 100% compatible with Pandoc Markdown but removes redundancy reducing the time to learn. (I work with many young people learning Pandoc and Markdown for the first time.)

Hopefully someone will find my fixes helpful and decide one day to implement them in whatever way seems best to them. Thanks for your consideration and time.

**Comment by alerque** [comment]
You are welcome to maintain your own fork with tweaks suitable for your usage. But at the same time we do still want to pull in the fixes and improvements you've presented here. The only request we made is that it not also mess with overriding themes. If you don't want to make that change to contribute here I'm happy to fix it up myself, and then you can fork with only a minimal set of changes including the theming bit.

**Comment by alerque** [comment]
(PR re-closed because I just realized it is against your master branch and I can't pull in the latest or fix stuff there, so I'll had to do this in another branch).
